ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -237,4 +237,33 @@ Public Class ucrFormDaily2
         Next
     End Sub
 
+    Public Sub checkTotal()
+        'Check total if required
+        ' Am not sure how to get this value yet
+        'If totalRequired = 1 Then
+
```

Comment:
This total required is a field in the obselement that am not sure currently how to get its value properly  @Patowhiz we can have a look at this together?

Reply:
Looking at this code we also need to think of how this can work in a really stable way.